docs/docs/administration-and-security/access-control/rbac.md

```diff
@@ -181,3 +185,117 @@ configure a role to create change requests only for features tagged with "market
 | Create change request    | Allows creating change requests for features in this environment.                                                        | Yes           |
 | Approve change request   | Allows approving or denying change requests in this environment.                                                         | Yes           |
 | View identities          | Grants read-only access to identities in this environment.                                                               |               |
+
+### Permission levels explained
+
+Some permissions are **project-level** and cannot be restricted to specific environments. This follows directly from the [Flagsmith data model](/flagsmith-concepts/data-model): features and segments are defined at the project level and shared across all environments.
+
+- **Create feature** — New features appear in all environments simultaneously
+- **Delete feature** — Removing a feature removes it from all environments simultaneously
+- **Manage segments** — Segments are project-wide and affect all environments
+
+To control **flag values** per environment, use environment-level permissions like _Update feature state_ and _Manage segment overrides_.
+
+## Common permission setups
+
+The following scenarios illustrate how to configure permissions for typical team structures. These examples help clarify which permissions to set at the project level versus the environment level.
+
+:::note
+
+These examples assume users have the built-in _User_ role at the organisation level (not _Organisation Administrator_). Organisation Administrators have full access to everything, so granular permissions only apply to users with the _User_ role.
+
+:::
+
+### Developer with Production restrictions
+
+**Goal**: Alice (developer) can create features and work freely in Development and Staging, but cannot modify Production directly — she must submit change requests instead.
+
+**Setup**:
+
+1. Create a _Developers_ group and add Alice to it
+2. Create a custom role called _Developer Access_ with these permissions:
+   - **Project-level**: View project, Create feature
+   - **Development environment**: Administrator
+   - **Staging environment**: Administrator
+   - **Production environment**: View environment, Create change request
```

Comment:
While this is probably a pretty pragmatic approach, I think a _strict_ separation of roles would probably result in something like the following _three_ roles: 

Non-prod Admin Role
Production Change Proposal Role
Base Project Access Role

I don't think we really need to change it for now though.